editor/src/components/inspector/inspector-strategies.tsx

```diff
@@ -62,9 +67,9 @@ export const updateFlexDirectionStrategies = (
       return null
     }
 
-    return elements.map((path) =>
+    return elements.flatMap((path) => [
```

Comment:
Isn't this unnecessary?

Reply:
thanks, fixed! (it got stuck there from an earlier iteration)